backend/app/Exceptions/TweetUpdateForbiddenException.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Exceptions;
+
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
+
+final class TweetUpdateForbiddenException extends AccessDeniedHttpException
```

Comment:
This is ok, but I would use only one `AuthorizationException` for all kinds of 403 errors.